domains/antonyhany.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "antonyhanyx1",
+        "email": "antonyhanywilliam2@gmail.com"
+    },
+    "record": {
```

```suggestion
    "records": {
```
it should be records, with "s"